src/ed_optimization/application/features/order/handlers/commands/process_order_command_handler.py

```
@@ -19,9 +19,21 @@
 
-    async def handle(self, request: ProcessOrderCommand) -> BaseResponse[None]: ...
+    async def handle(
+        self, request: ProcessOrderCommand) -> BaseResponse[None]: ...
 
-    def listener_callback(self, message: OrderModel) -> None:
-        print(message)
+    def _listener_callback(self, message: OrderModel) -> None:
+        LOG.info(f"order: {message}")
+
+        # Get all available delivery jobs
+        available_delivery_jobs = self._uow.delivery_job_repository.get_all(
```

Ruff (0.8.2)

Local variable `available_delivery_jobs` is assigned to but never used

Remove assignment to unused variable `available_delivery_jobs`

(F841)